app/controllers/kittens_controller.rb

```diff
@@ -7,4 +7,30 @@ def index
 
     render layout: true
   end
+
+  def new
+    # TODO
+    # @kitten = Kitten.new
+  end
+
+  def create
+    # TODO
+    # @kitten = Kitten.new
```

Comment:
If you have a form asking for the kitten's name, add a permitted param check to the class:


```
def permitted_params
  params.require(:kitten).permit(:name, :project_id)
end
```